File: src/main/java/net/atos/zac/zoeken/model/DatumRange.java
/*
 * SPDX-FileCopyrightText: 2022 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */

package net.atos.zac.zoeken.model;

import java.time.LocalDate;

public class DatumRange {
    public LocalDate van;

    public LocalDate tot;

    public DatumRange(final LocalDate van, final LocalDate tot) {
        this.van = van;
        this.tot = tot;
    }
}

File: src/main/app/src/app/zoeken/model/datum-range.ts
/*
 * SPDX-FileCopyrightText: 2022 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */

export class DatumRange {
  van: Date;
  tot: Date;

  constructor(van?: Date, tot?: Date) {
    this.van = van || null;
    this.tot = tot || null;
  }

  hasValue() {
    return this.van != null || this.tot != null;
  }
}
